config/config.conf

```diff
@@ -13,8 +13,29 @@ httpPort=7090
 # Metrics
 # ~~~~
 monitoringCluster="mad"
+#monitoringService="mad"
+#monitoringSinks=[
+#  {
+#    class = "com.arpnetworking.metrics.impl.ApacheHttpSink"
+#    bufferSize = 10000
+#    uri = "http://localhost:7090/metrics/v3/application"
+#    parallelism = 2
+#    maxBatchSize = 500
+#    emptyQueueInterval = "PT0.5S"
+#    eventsDroppedLoggingInterval = "PT1M"
+#    dispatchErrorLoggingInterval = "PT1M"
+#    unsupportedDataLoggingInterval = "PT1M"
+#  }
+#]
 #jvmMetricsCollectionInterval="PT1.0S"
 
+# NOTES:
+# - metricsClientHost and metricsClientPort are deprecated
+# - metricsClientHost and metricsClientPort no longer default from httpHost and httpPort
+# - you CANNOT specify either these AND monitoringSinks
```

Comment:
This wording is a bit awkward, specifically the "either and" construction.

Perhaps something more like "If either metricsClientHost or metricsClientPort are specified, then the legacy configuration will take precedence over monitoringSinks"

Reply:
Done